tests/ArchTest.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Geocodio\Tests;
+
+describe('Arch presets', function (): void {
+    arch('security preset')->preset()->security();
+    arch('php preset')->preset()->php();
+});
+
+describe('Custom relaxed preset', function (): void {
+    arch('No final classes')
+        ->expect('EchoLabs\Sparkle')
```

Comment:
Should this namespace be updated to `Geocodio`?

Reply:
Good catch, copypasta